perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1884,6 +1898,35 @@ sub user_hide_everywhere : Private {
     $c->stash->{status_message} = _('That user’s reports and updates have been hidden.');
 }
 
+sub send_login_email : Private {
+    my ( $self, $c, $email, $name, $user_id ) = @_;
+
+    my $token_data;
+    $token_data = {
+        email => $email,
+    };
+
+    if ( $user_id ) {
+        $token_data->{old_user_id} = $user_id;
+    }
+
+    $token_data->{name} = $name if $name;
+
+    my $token_obj = $c->model('DB::Token')->create({
```

Comment:
Could this and much of Auth.pm's `email_sign_in` be factored together, looks to basically be the same code, or not worth it?

Reply:
I'm not really sure it's worth it as the factoring out is going to add about as much extra code in to each method for the setup of the shared code as it saves I expect.